configs/14.0/deb/monolithic/control

```diff
@@ -154,7 +154,6 @@ Description: Advance Toolchain
 Package: advance-toolchain-libnxz
 Architecture: ppc64el
 Pre-Depends: advance-toolchain-__AT_MAJOR_INTERNAL__-libnxz (= __AT_FULL_VER__)
-Depends: ${shlibs:Depends} ${misc:Depends}
```

Comment:
Question for my understanding: I see that you removed the dependency only for one of the packages that were providing the conflicting files (as in #1553). The same for the cross case. How does that fix the issue?

Reply:
@mscastanho It's not helping to fix the issue. Actually I took advantage to improve the deb packaging of libnxz in AT. To make it clear, I updated the PR, instead of one commit, it has now 2 commits, one with the actual fix and one with some improvements.